packages/std/src/query/staking.rs

```diff
@@ -31,6 +31,70 @@ pub enum StakingQuery {
         /// The validator's address (e.g. (e.g. cosmosvaloper1...))
         address: String,
     },
+
+    /// Returns individual tokenize share record information by share by id
+    TokenizeShareRecordById { id: u64 },
+
+    /// Returns individual tokenize share record information by share denom
+    TokenizeShareRecordByDenom { denom: String },
+
+    /// Returns tokenize share records by address
+    TokenizeShareRecordsOwned { address: Addr },
+
+    /// Returns all tokenize share records
+    AllTokenizeShareRecords {},
+
+    /// Returns last tokenize share record id
+    LastTokenizeShareRecordId {},
+
+    /// Returns total tokenized staked assets
```

Comment:
I think the comment here should be more specific: does "total" mean "all tokenized assets that belong to a user"?

Reply:
Well, "total" means total :) **All** assets represented by tokenized shares

Reply:
wyell, tyotal myeans tyotal...